src/bedriftsmeny/Virksomhetsvelger/Menyvalg/Underenhetsvelger/Underenhetsvelger.less

```diff
@@ -61,30 +57,27 @@
     }
 
     &__menyvalg-wrapper {
-        &--lukket {
+        &--lukket,
+        &--apen {
             visibility: hidden;
+            transform: rotateX(-90deg);
+            transform-origin: top center;
+            opacity: 0.3;
+            transition: 0.3s ease, max-height 0.3s ease;
             max-height: 0;
-            opacity: 0;
-            transition: all 0.6s linear;
 
             .underenhet {
-                max-height: 0;
-                padding: 0 0.5rem 0 2.4rem;
-                transition: max-height 0.6s, padding 0.4s;
+                max-height: 20rem;
+                padding: 0.5rem 0.5rem 0.5rem 2.4rem;
             }
         }
 
         &--apen {
             visibility: visible;
-            max-height: 500rem;
+            max-height: 20rem;
```

Comment:
Det var en grunn for at jeg satte denne til 500rem, så dette må testes om funker hvis det er maaaange underenheter

Reply:
Det er et problem på ipad-visningen. Se gjerne på det :)

Reply:
Kanskje vi kan gjøre dropdown-menyen litt kortere på veldig liten skjerm? På Iphone SE er menyen litt for lang

Reply:
trenger vi egentlig ha max-height her i det heletatt? Når den er åpen så blir scorllbar aktivert, og det er definert max-height på parent container som plasser på høyden. Tenker linjen kan fjernes.

Reply:
Kan godt hende den kan fjernes ja :) Ser at det er et problem nå at innholdet i menyen flyter på utsiden av grensene sine, feks hvis man søker på "revisjon"

Reply:
Ser problemet som ble forklart her. Testet med Org som hadde 31 underenheter, da ble 20rem for lite. satt heller den til å holde max-height: max-content; Som lager like mye plass som underlisten skulle trenge.